modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/tree/io/BPlusMetaIO.java

```diff
@@ -328,10 +343,11 @@ public void setFlags(long pageAddr, boolean unwrappedPk, boolean inlineObjSuppor
     /**
      * @param pageAddr Page address.
      * @param inlineObjSupported Supports inline object flag.
+     * @param inlineDcSupported Supports decimal flag.
      * @param unwrappedPk Unwrap PK flag.
      * @param pageSize Page size.
      */
-    public static void upgradePageVersion(long pageAddr, boolean inlineObjSupported, boolean unwrappedPk, int pageSize) {
+    public static void upgradePageVersion(long pageAddr, boolean inlineObjSupported, boolean inlineDcSupported, boolean unwrappedPk, int pageSize) {
```

Comment:
is here any place or case when `inlineDcSupported` could be true? If asnwer is NO, seems we could remove the parameter at all.